lib/trino-filesystem-gcs/src/main/java/io/trino/filesystem/gcs/GcsFileSystemConfig.java

```diff
@@ -38,6 +43,16 @@ public class GcsFileSystemConfig
     private String jsonKey;
     private String jsonKeyFilePath;
 
+    // Note: without uniform strategy we cannot retry idempotent operations.
+    // The trino-filesystem api does not violate the conditions for idempotency, see https://cloud.google.com/storage/docs/retry-strategy#java for details.
+    private Optional<RetryStrategy> retryStrategy = Optional.of(RetryStrategy.UNIFORM);
```

Comment:
Let's remove this config. Users just care that it works, so we should make it work automatically. After reading that page, I'm still not sure why you would change it from UNIFORM to DEFAULT, so I doubt any users would either.